client/apps/admin-portal/src/app/manage/manage.component.html

```diff
@@ -1 +1,7 @@
-<p>manage works!</p>
+<div class="h-100 m-0">
```

Comment:
Not sure why this is here since the toolbar is going to be used throughout the app.
Let's put it in a place where it can be used once (maybe `app.component`?)

Also, the search bar is used only where there are sermons.